integr8ly/3scale-operator
# upgrade 3scale to 2.4

## Motivation
Upgrade 3Scale to version 2.4.0

## What
- Add the [template](https://github.com/3scale/3scale-amp-openshift-templates/blob/2.4.0.GA/amp/amp.yml) for 3Scale version 2.4.0
- Changed `system-provider-admin-route` to `system-provider-admin` to work with the new template.
- Changed the admin username to `admin` as the new template cannot accept usernames in an email format.
- Bump version to `0.0.5`

## Verification Steps
1. Run `make install` to install all required resources
2. Run `make deploy` to deploy the operator. Ensure that the operator points to this image: `quay.io/jameelb/3scale-operator:0.0.5`
3. Run `make create-examples` to create a sample 3scale custom resource
4. Ensure that the operator installs the correct version of 3scale `2.4.0`

## Checklist:
